api/src/helper/lib/base-nlp-helper.ts

```
@@ -225,4 +225,145 @@ export default abstract class BaseNlpHelper<
+
+          // Apply preprocessing if needed
+          if (patternValue.metadata?.removeSpaces) {
+            value = value.replace(/\s+/g, '');
+          }
+
+          if (patternValue.metadata?.toLowerCase) {
+            value = value.toLowerCase();
+          }
+
+          if (patternValue.metadata?.stripDiacritics) {
+            value = value.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)